cves/kernel/CVE-2017-5576.yml

```diff
@@ -448,7 +455,10 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+   This issue was caused by a race condition that was not checked for, as well
```

Comment:
It might be helpful if you specified what kind of race condition was overlooked.